src/main/res/values/strings.xml

```diff
@@ -166,22 +166,22 @@
     <string name="setting_minimum_sync_chars" translatable="false">minimum_sync_chars</string>
 
     <string name="pref_push_on_receive">Push SMS on reception</string>
-    <string name="pref_sync_wifi">Synchronize in Wi-Fi</string>
-    <string name="pref_sync_4g">Synchronize in 4G</string>
-    <string name="pref_sync_3g">Synchronize in 3G</string>
-    <string name="pref_sync_gprs">Synchronize in 2.5G (GPRS)</string>
-    <string name="pref_sync_2g">Synchronize in 2G</string>
+    <string name="pref_sync_wifi">Synchronize on Wi-Fi</string>
+    <string name="pref_sync_4g">Synchronize on 4G</string>
+    <string name="pref_sync_3g">Synchronize on 3G</string>
+    <string name="pref_sync_gprs">Synchronize on 2.5G (GPRS)</string>
+    <string name="pref_sync_2g">Synchronize on 2G</string>
     <string name="pref_sync_others">Synchronize in other modes</string>
     <string name="title_activity_login">Sign in</string>
 
     <!-- Login -->
-    <string name="prompt_login">Login</string>
+    <string name="prompt_login">Log in</string>
```

Comment:
sorry but i'm not in favor of this space

Reply:
"Login" is not a verb, whereas "Log in" is.